x/rewards/keeper/state_flatfee.go

```diff
@@ -44,3 +47,39 @@ func (s FlatFeeState) RemoveFee(contractAddr sdk.AccAddress) {
 	store := prefix.NewStore(s.stateStore, types.FlatFeePrefix)
 	store.Delete(contractAddr.Bytes())
 }
+
+// Import initializes state from the genesis flat fees data.
+func (s FlatFeeState) Import(flatFees []types.FlatFee) {
+	for _, flatFee := range flatFees {
+		contractAddr := flatFee.MustGetContractAddress()
+		fee := flatFee.GetFlatFee()
+
+		if !fee.Amount.IsPositive() {
+			panic(fmt.Sprintf("flat fee: %+v is invalid, err: %s", flatFee, sdkErrors.ErrInvalidCoins))
```

Comment:
move this check to validate genesis too

Reply:
Can remove this check, as fee is being validated in genesis https://github.com/archway-network/archway/pull/275/files#diff-805f961f8d4413593130e2fd0cb15f43fa255fbc70caadfb82f33ed50f9da03aR122

Reply:
Removed this check dc9789782f989062b2ba8d2609a531e97fb6ff20